AppCenterAnalytics/AppCenterAnalytics/Internals/Session/MSACSessionTracker.h

```diff
@@ -46,6 +46,16 @@ NS_ASSUME_NONNULL_BEGIN
  */
 - (void)stop;
 
+/**
+ * Manual session tracking checking.
+ */
+- (void)enableManualSessionTracker;
+
+/**
+ * Start manual session tracking.
```

Comment:
`Setting the state of manually session generation process.` -> `Enable manual session tracker.`